pkgs/apps/gamess-us/default.nix

```diff
@@ -1,5 +1,7 @@
 { stdenv, lib, makeWrapper, fetchFromGitLab, requireFile, gfortran, writeTextFile, cmake, perl
 , tcsh, mpi, blas, hostname, openssh, gnused, libxc, ncurses
+, enableMpi ? true
+, scratchHere ? false
```

Comment:
Instead of creating this special option for pysisyphus, wouldn't it be simpler if pysisyphus manipulates `$SCRATCH` instead before calling gamess?

Reply:
You're right. I've changed it. Much better.